### PR TITLE
[No ticket] Mege #2145 from main repo.

### PR DESCRIPTION
**Pull request by AbnerZheng**
### Overview
The first commit of this PR merges the https://github.com/CasperLabs/CasperLabs/pull/2145 from the main repo, and the remaining commits rename `public key hash` to `account hash` where are not fixed by that PR. 

### Which JIRA ticket does this PR relate to?
No ticket

### Complete this checklist before you submit this PR
- [X] This PR contains no more than 200 lines of code, excluding test code.
- [X] This PR meets [CasperLabs coding standards](https://casperlabs.atlassian.net/wiki/spaces/EN/pages/16842753/Coding+Standards).
- [ ] If this PR adds a new feature, it includes tests related to this feature.
- [X] You assigned one person to review this PR.
- [X] Your GitHub account is linked with our [Drone CI](https://drone-auto.casperlabs.io/) system. This is necessary to run tests on this PR.
- [X] Do not forget to run `bors r+` if GitHub policy is not enforced, e.g. when merging into another feature branch. It may be omitted under some circumstances if this PR intentionally assumes that integration tests will fail but will be fixed with the future PRs.

### Notes
_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._

### CODE REVIEW

**Comment by AbnerZheng** [comment]
@piotr-dziubecki ^

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "packages/*"
   ],
   "devDependencies": {
-    "lerna": "^3.22.1",
-    "concurrently": "^3.6.0"
+    "concurrently": "^3.6.0",
+    "husky": "^4.2.5",
```
Comment (PfanP) [inline package.json]: Husky is installed but I can't find the script for pre-commit. Are we going to do this on the next pr?

**Review by PfanP — APPROVED** [review]

**Comment by AbnerZheng** [comment]
bors cancel

**Pull request merged**